neuron_morphology/pipeline/post_data_to_s3.py

```diff
@@ -0,0 +1,145 @@
+import os
+import boto3
+import logging
+import json
+from botocore.exceptions import ClientError
+from zipfile import ZipFile, ZIP_DEFLATED
+from io import BytesIO
+from argschema import ArgSchemaParser
+from neuron_morphology.pipeline._schemas import InputParameters
+
+def post_file_to_s3(file_name, bucket_name):
+    """
+    This is to post data to S3 bucket
+    
+    Parameters
+    ------------------
+    file_name: the file need to be uploaded
+    bucket_name: s3 bucket name
+    
+    Return
+    ------------------
+    True if successful
+
+    """
+    aws_id = os.getenv('aws_access_key_id')
+    aws_key = os.getenv('aws_secret_access_key')
+    
+    try:
+        s3_client = boto3.client(
+            's3',
+            aws_access_key_id= aws_id,
+            aws_secret_access_key= aws_key
+        )
+
+        response = s3_client.upload_file(file_name, bucket_name, file_name)
+
+    except ClientError as e:
+        logging.error(e)
```

Comment:
Why are you handling this error here? This is not a recoverable situation.

Reply:
removed